component/mcsd/test/org1_dir_history_response-page2.json

```diff
@@ -6,7 +6,7 @@
   "type": "history",
   "entry": [
     {
-      "fullUrl": "{{ORG1_DIR_BASEURL}}/Endpoint/ep-2",
+      "fullUrl": "Endpoint/ep-2",
```

Comment:
Maybe we can change the page 2 fullUrls to version-specific URLs (relative and absolute) so we ensure that also keeps working